src/loadgenerator/Dockerfile

```diff
@@ -24,10 +24,11 @@ RUN pip install -r requirements.txt
 
 # Copy tasks and runner script
 COPY run.sh .
-COPY locust-tasks ./locust-tasks
+COPY *.py ./
+COPY locust_tasks ./locust_tasks
 
 # Expose the required Locust ports
-EXPOSE 5557 5558 8089
+EXPOSE 5557 5558 8089 8090
```

Comment:
curious, what is this port used for (8090)? Perhaps, worth adding a comment.

Reply:
Sounds good. I added more documentation for the original ports exposed here as well